modules/mod_ginger_base/templates/media/media.audio.tpl

```diff
@@ -5,19 +5,15 @@ as
 %}
 
 <div class="{{ class }}">
-
     <audio controls>
         <source src="/media/attachment/{{ id.medium.filename }}" type="{{ id.medium.mime }}">
     </audio>
 
-    <div>
-        {% if id.summary %}
-            {{ id.summary }}
-        {% elif id.title %}
-            {{ id.title }}
-        {% endif %}
-    </div>
-
+    {% if caption|default:m.rsc[id].summary|default:id.title as caption %}
```

Comment:
Why `m.rsc[id].summary` but `id.title`?

Reply:
Good question, changed this to the same as title